scraper/Dockerfile

```diff
@@ -10,5 +10,6 @@ FROM alpine
 RUN apk --no-cache add ca-certificates
 RUN mkdir app
 COPY --from=builder /go/src/github.com/codeuniversity/smag-mvp/instascraper_scraper /app
+COPY --from=builder /go/src/github.com/codeuniversity/smag-mvp/useragents.json /app
```

Comment:
You don't need the `--from=builder` 
I think the caching is simpler if you just use the useragents.json from the build-context

Reply:
I will create another PR for that